packages/auth/src/grant/service.ts

```diff
@@ -89,6 +91,15 @@ async function issueGrant(
   })
 }
 
+async function denyGrant(
+  deps: ServiceDependencies,
+  grantId: string
+): Promise<Grant> {
```

Comment:
```suggestion
): Promise<void | Grant> {
```
and test that `undefined` is returned for unknown `grantId`

Reply:
Might `null` work better? `void` poses a bit of an issue with typings

Reply:
:+1: `getByContinue`'s also `Promise<Grant | null>`